feature-flags.json

```diff
@@ -0,0 +1,39 @@
+{
+  "features": [
+    {
+      "name": "sms",
+      "description": "Sends reminder text messages",
+      "enabled_for": {
+        "environments": ["staging", "production"]
+      }
+    },
+    {
+      "name": "capped_bookings",
+      "description": "Allows schools to provide a maximum number of bookings per date",
+      "enabled_for": {
+        "environments": ["development", "test", "servertest"]
+      }
+    },
+    {
+      "name": "reminders",
+      "description": "Sends reminder emails and text messages",
+      "enabled_for": {
+        "environments": ["development", "test", "servertest", "staging", "production"]
```

Comment:
Are the changes in here intentional? If I'm reading it right:

reminders - now enabled for dev
candidates_dashboard - now enabled for dev/test/servertest

Reply:
Removed development from reminders, but the candidates dashboard was using the `phase` var which enabled it in dev, test, and servertest